k8s/service_controller/server/k8s/storage.go

```diff
@@ -29,10 +29,11 @@ func CreateServiceStorage() server.ServiceStorage {
 // listSB is only used for unmarshalling the list of service brokers
 // for returning to the client
 type listSB struct {
-	Items []*model.ServiceBroker
+	Items []k8sServiceBroker `json:"items"`
```

Comment:
That seems to work too. Not sure how big of a difference it is in this case.

(also, it's a *slice* of whatevers)